### PR TITLE
Updated Matrix4x4.cs

### PR DESCRIPTION
**Pull request by aakashapoorv**
Implemented GetColumn, GetRow, SetColumn and SetRow.

### CODE REVIEW

File: Framework/PressPlay.FFWD/Matrix4x4.cs [hunk 1]
```diff
@@ -232,22 +248,103 @@ public bool Equals(Matrix4x4 other)
...
+                    m33 = v.w;
+                    break:
+                default:
+                    throw new IndexOutOfRangeException("i must be from 0 to 3 inclusive.");
+            }
         }
 
         public void SetRow(int i, Vector4 v)
         {
-            throw new NotImplementedException();
```
Comment (aakashapoorv) [inline Framework/PressPlay.FFWD/Matrix4x4.cs:250]: From: http://docs.unity3d.com/Documentation/ScriptReference/Matrix4x4.SetRow.html

void SetRow(int i, Vector4 v);
Description
Sets a row of the matrix.

The i-th row is set from v. i must be from 0 to 3 inclusive.

File: Framework/PressPlay.FFWD/Matrix4x4.cs [hunk 2]
```diff
@@ -232,22 +248,103 @@ public bool Equals(Matrix4x4 other)
...
+                case 3:
+                    return Vector4(m03,m13,m23,m33);
+                default:
+                    throw new IndexOutOfRangeException("i must be from 0 to 3 inclusive.");
+            }
         }
 
         public void SetColumn(int i, Vector4 v)
         {
-            throw new NotImplementedException();
```
Comment (aakashapoorv) [inline Framework/PressPlay.FFWD/Matrix4x4.cs:245]: From: http://docs.unity3d.com/Documentation/ScriptReference/Matrix4x4.SetColumn.html

void SetColumn(int i, Vector4 v);
Description
Sets a column of the matrix.

The i-th row is set from v. i must be from 0 to 3 inclusive.

File: Framework/PressPlay.FFWD/Matrix4x4.cs [hunk 3]
```diff
@@ -232,22 +248,103 @@ public bool Equals(Matrix4x4 other)
...
+                    return Vector4(m30,m31,m32,m33);
+                default:
+                    throw new IndexOutOfRangeException("i must be from 0 to 3 inclusive.");
+            }
+
         }
 
         public Vector4 GetRow(int i)
         {
-            throw new NotImplementedException();
```
Comment (aakashapoorv) [inline Framework/PressPlay.FFWD/Matrix4x4.cs:240]: From: http://docs.unity3d.com/Documentation/ScriptReference/Matrix4x4.GetRow.html

Vector4 GetRow(int i);
Description
Returns a row of the matrix.

The i-th row is returned as a Vector4. i must be from 0 to 3 inclusive.

File: Framework/PressPlay.FFWD/Matrix4x4.cs [hunk 4]
```diff
@@ -232,22 +248,103 @@ public bool Equals(Matrix4x4 other)
 
         public Vector4 GetColumn(int i)
         {
-            throw new NotImplementedException();
```
Comment (aakashapoorv) [inline Framework/PressPlay.FFWD/Matrix4x4.cs:235]: From: http://docs.unity3d.com/Documentation/ScriptReference/Matrix4x4.GetColumn.html

Vector4 GetColumn(int i);
Description
Get a column of the matrix.

The i-th column is returned as a Vector4. i must be from 0 to 3 inclusive.

File: Framework/PressPlay.FFWD/Matrix4x4.cs [hunk 5]
```diff
@@ -90,127 +90,143 @@ public static Matrix4x4 zero
...
+        public Matrix4x4 inverse
+        {
+            get
+            {
+                return Matrix4x4.Inverse(this);
+            }
+        }
+
+        public bool isIdentity(Matrix4x4 m)
+        {
```
Comment (aakashapoorv) [inline Framework/PressPlay.FFWD/Matrix4x4.cs:104]: From: http://docs.unity3d.com/Documentation/ScriptReference/Matrix4x4-isIdentity.html
bool isIdentity;
Description
Is this the identity matrix?

**Comment by fehaar** [comment]
Thanks! Great Work.

**Pull request merged**